strapi/strapi
# feat(main-nav): Main nav refactoring, change links order

### What does it do?

It changes the main nav links order based on the specifications on the figma file

I added a position property, which is a numeric and can help us to reorder the plugins.
The links in the main nav are all sorted by position starting from 0 to the last one, the links with the same position number are sorted by link label
If you don't specify the link position then its position is in the middle of the main nav, before the plugin list

### Why is it needed?

To give more visibility to the most used links

### How to test it?

Just check the new main nav and check the figma file (you can find it in the jira ticket)

### Related issue(s)/PR(s)

CS-768
